### DISCUSSION TITLE
courses/beginners-web-handbook/google-project-1/10-search-input

### DISCUSSION THREAD

**Discussion by giscus** [crwdu/lms-contents-js101#29] [Q&A]
# courses/beginners-web-handbook/google-project-1/10-search-input

바닐라코딩

https://www.vcdr.club/courses/beginners-web-handbook/google-project-1/10-search-input

**Comment by junpil-TnT** [comment]
2.입력창 중앙 배치하기
input 요소의 부모 요소에게 .input-box 라는 class 이름을 지정해주고, 아래와 같이 CSS를 추가해보세요.
이부분에서 css를 추가해도 중앙으로 이동되지 않습니다

혹시 입력창부분에 오류가 있나싶어 몇번 더 확인 해봐도 이동이 안되네요

**Reply by junpil-TnT** [reply]
<html lang="ko">
    <head>
        <title>바닐라 코딩</title>
        <style>
            body {
                text-align: right;
            }

            a {
                margin: 10px;
                font-size: 14px;
            }

            .sign-in {
                background-color: #1a73e8;
                color: white;
                padding: 9px 23px;
                border-radius: 4px;
            }

            header {
                margin-top: 20px;
            }

            .logo-box {
                text-align: center;
                margin-top: 100px;
            }

            .logo-box img {
                width: 200px;
            }

    input {
        border-radius: 50px;
        border: 1px solid gray; // [설명] 1px의 두께, 실선, 회색
    }

    .input-box {
        text-align: center;
    }
</style>
    </head>
    <body>
        <header><a>Gmail</a><a>Images</a><a class="sign-in">Sign-in</a></header>

        <div class="logo-box">
            <img src="logo.jpeg" />
        </div>

        <div>
            <input type="text" />
        </div>

        <div><a>Google Search</a> <a>I'm feeling Lucky</a></div>

        <div>Google offered in: 한국어</div>

        <footer>Footer</footer>
    </body>
</html>

**Comment by leo-vanilla** [comment]
안녕하세요. 🙂

작성해주신 코드를 살펴보니 CSS 파트에는 `.input-box` 클래스를 잘 작성해주신 것 같아요.

다만, HTML 코드에 말씀하신 input 요소의 부모 요소인 div 태그에 `.input-box` 라는 클래스가 정상적으로 작성되지 않은 것 같아요.

바로 위의 `.logo-box`의 케이스를 참고하셔서 한번 작업해보시면 좋을 것 같습니다!

한번 시도해신 뒤에도 잘 안되시면 다시 질문 남겨주세요.

감사합니다. 🙇🏻‍♂️

**Reply by junpil-TnT** [reply]
친절한 답변 너무 감사드립니다!
HTML 코드에        
 <div class="input-box">
 <input type="text" />
  </div>
를 추가한다음 작업하니 완료됐습니다!
조언 너무 감사드립니다

**Reply by junpil-TnT** [reply]
<div class="input-box">
            <input type="text" />
        </div> <<<<<이게 댓글 내용에 빠졌네요
